core/modules/menu/js/menu.admin.js

```diff
@@ -19,6 +19,60 @@ Backdrop.behaviors.menuAdminFieldsetSummaries = {
   }
 };
 
+Backdrop.behaviors.menu_edit_item_parents = {
+  attach: function (context, settings) {
+    // Get the menu parent options from settings.
+    var menuOptions = settings.menu_edit_item_parents;
+    // Get the current menu name from settings.
+    var menuName = settings.menu_edit_item_menu;
+    // Always move the menu selector before the menu parent selector.
+    $('.form-item-parent-menu').insertBefore('#menu-parent-select-wrapper');
+    // On load set the current menu as default in the menu select.
+    Backdrop.menu_edit_update_parent_list(menuName);
+    // Ensure that the menu parent select list is filtered again when any ajax
+    // process runs.
+    $( document ).ajaxComplete(function( event, xhr, settings ) {
+      if (settings.url=='/system/ajax') {
```

Comment:
Code syntax suggestion... but really this might need to be fixed so that it works both without clean URLs and with a different base URL? Maybe use `Backdrop.relativeUrl('system/ajax')` instead here.

```suggestion
      if (settings.url == '/system/ajax') {
```

Reply:
Fixed